inc/composer/class-command.php

```diff
@@ -175,6 +200,21 @@ protected function get_env() : array {
 	protected function start( InputInterface $input, OutputInterface $output ) {
 		$output->writeln( '<info>Starting...</>' );
 
+		if ( $input->hasParameterOption( '--docker-sync' ) || $this->has_docker_sync_file() ) {
+			$sync = new Process( 'docker-sync start', 'vendor' );
+			// Might need to download the docker-sync container so extend this beyond default 60s.
+			$sync->setTimeout( 1200 );
+			$sync->setTty( true );
+			$sync_failed = $sync->run( function ( $type, $buffer ) {
```

Comment:
I guess $buffer is a string? Can we type hint them both?

Reply:
Maybe. We haven't type hinted it anywhere else in the file but could do.